README.md

```diff
@@ -33,5 +35,6 @@ Copy the code already present, paste it and make the following changes.
 
 ### OUR VALUABLE CONTRIBUTORS✨
 <a href="https://github.com/ShehrozIrfan/hacktoberfest-2022-first-contribution/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=ShehrozIrfan/hacktoberfest-2022-first-contribution" />
+  <img src="https://contrib.rocks/image?repo=ShehrozIrf![Uploading image.png…]()
```

Comment:
@Shaurya50211 
I think it won't work. 
Fix it.

Reply:
Should I remove those images?

Reply:
No. Don't remove.
I merged your changes, but not unable to see any changes.
You can take the latest pull from master. And check is it working alright?